content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -91,6 +91,10 @@ For example, the line below states that the task must be started every Friday at
 
 To generate CronJob schedule expressions, you can also use web tools like [crontab.guru](https://crontab.guru/).
 
+In rare cases, when it is necessary to create a job, that should be triggered _only_ manually, and never - automatically, use 31st of February:
```

Comment:
Why would you do it this way, rather than creating a [suspended](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-suspension) CronJob and then just use `kubectl create job --from <cronjob name>` ?